src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const carCosts = 40;
+
+  for (let i = 0; i <= days; i++) {
+    result = days * carCosts;
+  }
```

Comment:
this loop does nothing
the final `result` value will be `days * carCosts` anyway